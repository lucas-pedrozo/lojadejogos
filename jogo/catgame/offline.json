{"version":1716248423332,"fileList":["scripts/c3runtime.js","data.json","style.css","scripts/offlineclient.js","images/shared-0-sheet4.webp","images/planodefundoemblocos-sheet0.webp","images/shared-0-sheet0.webp","images/shared-0-sheet1.webp","images/shared-0-sheet3.webp","images/shared-0-sheet2.webp","scripts/opus.wasm.js","scripts/opus.wasm.wasm","scripts/main.js","workermain.js","scripts/dispatchworker.js","scripts/jobworker.js","scripts/supportcheck.js","icons/icon-16.png","icons/icon-32.png","icons/icon-64.png","icons/icon-256.png","icons/icon-128.png","icons/loading-logo.png","icons/icon-512.png","media/Som de pulo.webm","media/Som andando.webm","media/Esplosao.webm","media/Bossa nova.webm","scripts/register-sw.js"]}