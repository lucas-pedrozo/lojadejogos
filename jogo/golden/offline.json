{"version":1713190333325,"fileList":["scripts/c3runtime.js","data.json","style.css","scripts/offlineclient.js","images/shared-0-sheet4.webp","images/shared-0-sheet5.webp","images/shared-0-sheet1.webp","images/shared-0-sheet3.webp","images/shared-0-sheet6.webp","images/planodefundoemblocos5-sheet0.webp","images/shared-0-sheet8.webp","images/shared-0-sheet11.webp","images/shared-0-sheet9.webp","images/planodefundoemblocos6-sheet0.webp","images/planodefundoemblocos-sheet0.webp","images/planodefundoemblocos2-sheet0.webp","images/planodefundoemblocos3-sheet0.webp","images/shared-0-sheet10.webp","images/shared-0-sheet7.webp","images/shared-0-sheet2.webp","images/shared-0-sheet0.webp","scripts/opus.wasm.js","scripts/opus.wasm.wasm","scripts/main.js","workermain.js","scripts/dispatchworker.js","scripts/jobworker.js","scripts/supportcheck.js","icons/icon-16.png","icons/icon-32.png","icons/icon-64.png","icons/icon-128.png","icons/icon-256.png","icons/icon-512.png","icons/loading-logo.png","media/cork-85200.webm","media/cote.webm","media/Ding - Copia.webm","media/agua.webm","media/FIRE.webm","media/fiere2.webm","media/music.webm","media/bone.webm","media/Daisies(MP3_160K).webm","scripts/register-sw.js"]}